corehq/apps/auditcare/migrations/0004_add_couch_id.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.20 on 2021-05-03 22:46
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('auditcare', '0003_truncatechars'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='accessaudit',
+            name='couch_id',
+            field=models.CharField(db_index=True, max_length=126, null=True),
+        ),
+        migrations.AddField(
+            model_name='navigationeventaudit',
+            name='couch_id',
+            field=models.CharField(db_index=True, max_length=126, null=True),
```

Comment:
Also creating the index like this will create a  pattern_ops index as well which we don't want

Reply:
One reason we might want the index prior to the migration that it will speed up the lookups dramatically which will almost certainly outweigh the slow down from keeping the index up to date

Reply:
Thanks, opened https://github.com/dimagi/commcare-hq/pull/29780

Reply:
Also rebased this PR onto https://github.com/dimagi/commcare-hq/pull/29780

Reply:
(no changes to individual commits other than removing the couch_id attr from models.py and the migration file, which are now part of the new PR)